src/views/EditorView/TopNavigationBar/TopNavigationBar.tsx

```diff
@@ -96,8 +96,8 @@ const TopNavigationBar: React.FC<IProps> = (props) => {
                     <div className="Header" onClick={closePopup}>
                         <img
                             draggable={false}
-                            alt={'showniq icon'}
-                            src={'/showniq-icon.png'}
+                            alt={'make-sense'}
```

Comment:
[major]
This change seems to discard the showniq logo and use the old make-sense logo. Is this your intention?